src/server/terrain/simple_structures/SbbGen.zig

```diff
@@ -18,37 +18,59 @@ pub const generationMode = .floor;
 
 const SbbGen = @This();
 
-structureRef: *const sbb.StructureBuildingBlock,
+structureRef: ?*const sbb.StructureBuildingBlock,
```

Comment:
Why is this optional now?

Reply:
To allow for dummy generators if generator specified in configuration doesn't exist instead of panicking.

Reply:
Well, but you know my opinion on this: These problems should be resolved at load time, instead of checking an extra condition on every structure placed.

Reply:
It is not a mindset I want to have here.
Being afraid that everything is going to break some API is not something we should have to worry about.
Being able to move fast and break things is important. In my opinion backwards compatibility is burden we should not carry unless it's necessary (and the one and only place where it is necessary is in storage).

Reply:
Ok. Then because it requires changes in many files (all of the generators) I won't be doing that in this PR. I will just use `panicWithMessage` instead.